app/static/js/userProfile.js

```diff
@@ -140,9 +141,25 @@ function displayMessage(message, color) {  // displays message for saving backgr
     setTimeout(function() {$("#displaySave").html("").removeClass("text-"+ color)}, 2000)
 }
 
-function updateManagers(el, volunteer_username ){// retrieve the data of the studnet staff and program id if the boxes are checked or not
-  var program_id=$(el).attr('data-programid');
-  action= el.checked ? 'add' : 'remove';
+$(function() {
+     toastElList = [].slice.call(document.querySelectorAll('.toast'))
```

Comment:
Could I get a short reason as to the name of toastElList?

Reply:
Will rename it to a more appropriate name.